src/effects/effectrack.cpp

```diff
@@ -296,57 +384,19 @@ QuickEffectRack::QuickEffectRack(EffectsManager* pEffectsManager,
 }
 
 void QuickEffectRack::configureEffectChainSlotForGroup(
-        EffectChainSlotPointer pSlot, const ChannelHandleAndGroup& handle_group) {
-    // Register this channel alone with the chain slot.
-    pSlot->registerChannel(handle_group);
-
-    // Add a single EffectSlot for the quick effect.
-    pSlot->addEffectSlot(QuickEffectRack::formatEffectSlotGroupString(
-            getRackNumber(), handle_group.name()));
-
-    // TODO(rryan): Set up next/prev signals.
-
-    // Now load an empty effect chain into the slot so that users can edit
-    // effect slots on the fly without having to load a chain.
-    EffectChainPointer pChain = pSlot->getOrCreateEffectChain(m_pEffectsManager);
-
-    // Enable the chain for the channel by default.
-    pChain->enableForChannel(handle_group);
-
-    // Set the chain to be fully wet.
-    pChain->setMix(1.0);
```

Comment:
Is this lost?

Reply:
No, it is moved to PerGroupRack::setupForGroup

Reply:
Ok, that makes sense.